metals/src/main/scala/scala/meta/internal/worksheets/WorksheetClassLoader.scala

```diff
@@ -0,0 +1,15 @@
+package scala.meta.internal.worksheets
+
+/**
+ * ClassLoader that is used to reflectively invoke mdoc for worksheet evaluation.
+ */
+class MdocClassLoader(parent: ClassLoader) extends ClassLoader(null) {
```

Comment:
Why do we need this classloader? It just loads mdoc, but aside from filtering doesn't do anything.

Reply:
I updated the docstring to explain why this classloader is necessary, it isolates the mdoc classloader from the metals classloader so that we can call mdoc 2.11/2.13 from the metals 2.12 server.